mmv1/products/developerconnect/Connection.yaml

```diff
@@ -12,205 +12,381 @@
+update_verb: PATCH
+id_format: projects/{{project}}/locations/{{location}}/connections/{{connection_id}}
 import_format:
-  - 'projects/{{project}}/locations/{{location}}/connections/{{connection_id}}'
-timeouts:
-  insert_minutes: 20
-  update_minutes: 20
-  delete_minutes: 20
-autogen_async: true
-async:
-  actions: ['create', 'delete', 'update']
-  type: 'OpAsync'
-  operation:
-    base_url: '{{op_id}}'
-  result:
-    resource_inside_response: true
-custom_code:
+  - projects/{{project}}/locations/{{location}}/connections/{{connection_id}}
 examples:
-  - name: 'developer_connect_connection_basic'
+  - name: 'developer_connect_connection_github'
     primary_resource_id: 'my-connection'
     primary_resource_name: 'fmt.Sprintf("tf-test-connection%s", context["random_suffix"])'
-    min_version: 'beta'
     vars:
       connection_name: 'tf-test-connection'
   - name: 'developer_connect_connection_github_doc'
-    min_version: 'beta'
     exclude_test: true
+  - name: 'developer_connect_connection_github_enterprise'
+    primary_resource_id: 'my-connection'
+    primary_resource_name: 'fmt.Sprintf("tf-test-connection%s", context["random_suffix"])'
+    vars:
+      connection_name: 'tf-test-connection'
+  - name: 'developer_connect_connection_github_enterprise_doc'
+    exclude_test: true
+  - name: 'developer_connect_connection_gitlab'
+    primary_resource_id: 'my-connection'
+    primary_resource_name: 'fmt.Sprintf("tf-test-connection%s", context["random_suffix"])'
+    vars:
+      connection_name: 'tf-test-connection'
+  - name: 'developer_connect_connection_gitlab_doc'
+    exclude_test: true
+  - name: 'developer_connect_connection_gitlab_enterprise'
+    primary_resource_id: 'my-connection'
+    primary_resource_name: 'fmt.Sprintf("tf-test-connection%s", context["random_suffix"])'
+    vars:
+      connection_name: 'tf-test-connection'
+  - name: 'developer_connect_connection_gitlab_enterprise_doc'
+    exclude_test: true
+autogen_async: true
+async:
+  operation:
+    timeouts:
+      insert_minutes: 20
+      update_minutes: 20
+      delete_minutes: 20
+    base_url: '{{op_id}}'
+  actions:
+    - create
+    - delete
+    - update
+  type: OpAsync
+  result:
+    resource_inside_response: true
+  error: {}
+  include_project: false
+autogen_status: Q29ubmVjdGlvbg==
 parameters:
-  - name: 'location'
+  - name: location
     type: String
-    description: |
-      Resource ID segment making up resource `name`. It identifies the resource
-      within its parent collection as described in https://google.aip.dev/122. See documentation
-      for resource type `developerconnect.googleapis.com/GitRepositoryLink`.
-    min_version: 'beta'
+    description: Resource ID segment making up resource `name`. It identifies the resource within its parent collection as described in https://google.aip.dev/122.
+    immutable: true
     url_param_only: true
     required: true
-    immutable: true
-  - name: 'connectionId'
```

Comment:
I wonder what is the reason the change the format of fields, for example, removing quotes? Are the yaml files generated by autogen?

Reply:
yeah i kept the format the same after generating the yaml files using tf autogen